### ISSUE TITLE
Reason for commented-out test?

### ISSUE THREAD

**Issue by svdo** [Mocolate/reinforcement_learning_pacman#2]
I noticed that this test is commented out. I'm not sure what the reason is, but I suggest either activating the test, or removing it altogether, in order to avoid confusion.

https://github.com/Mocolate/reinforcement_learning_pacman/blob/9332e36f9c11603ef4be1d125ff8de097a628ed5/reinforcement/tests/test_qlearning_agents_step1.py#L22-L25

**Comment by Mocolate** [comment]
Yes, good question. This test was added to ensure that the `set_q_value` method only takes floats and raises an error otherwise. 
This caused some issues the previous time we ran this workshop, as this requirement was not mentioned in the assignment. So either we remove this test entirely or put the requirement in the assignment. 

Personally I'd prefer to just remove the test entirely, as the goal of this workshop is the reinforcement learning and not type-testing in python. What do you think?

**Comment by svdo** [comment]
Yeah I’d say removing it sounds good. 👍
